platform-sdk/swirlds-state-impl/src/main/java/com/swirlds/state/merkle/MerkleStateRoot.java

```diff
@@ -1238,9 +1244,11 @@ private void migrateKVStates(
                             older.release();
                             virtualMapRef.set(currentMap);
                         }
-                        final var keyBytes = Bytes.wrap(createVirtualMapKeyBytesForKV(
+                        final Bytes keyBytes = Bytes.wrap(createVirtualMapBytesForKV(
                                 serviceName, stateKey, pair.key().toByteArray()));
```

Comment:
Both for keys and values, there is no need to copy their Bytes to a byte array, which is then copied to a byte buffer in `createVirtualMapBytesKV()` These Bytes objects can be used as is

Reply:
Done